recipe/run_test.sh

```diff
@@ -9,7 +9,7 @@ if [ "${JAVA_HOME}" != "${PREFIX}/lib/jvm" ] && [ "${JAVA_HOME}" != "${PREFIX}/L
 fi
 
 pushd test-nio
-  javac TestFilePaths.java
+  javac -verbose TestFilePaths.java || cat *.log
```

Comment:
Do you need this still or was it for debugging?

Reply:
can be reverted

```suggestion
  javac -verbose TestFilePaths.java
```